lib/rabbitmq/cli/ctl/commands/exec_command.ex

```diff
@@ -46,6 +44,8 @@ defmodule RabbitMQ.CLI.Ctl.Commands.ExecCommand do
     end
   end
 
+  use RabbitMQ.CLI.Core.RequiresRabbitAppRunning
```

Comment:
The exec command does not need the rabbit app to be running. It does not even need a node to be running